vendor/engines/secure_rooms/config/locales/en.yml

```diff
@@ -55,6 +55,9 @@ en:
     attributes:
       user:
         card_number: Card Number
+      order_detail/occupancy:
+        exit_at: Time of Exit
+        actual_duration_mins: Duration
```

Comment:
It's for nested attributes error messages. Not sure why it couldn't pick up the normal version, but the messages were like "Occupancy exit at cannot be blank".